ew_agents/agent.py

```
@@ -293,12 +498,119 @@ def create_agent(config):
+            list(knowledge_tools_wrapped.keys())
+        ))
+    }
+
+# Clean exports
 __all__ = [
+    # Core agent exports
     'root_agent', 
     'coordinator_agent',
     'data_eng_agent',
     'osint_agent', 
     'lexicon_agent',
```

Ruff (0.12.2)

Undefined name `lexicon_agent` in `__all__`

(F822)